app/models/campaign.js

```diff
@@ -18,6 +18,27 @@ var CampaignSchema = new Schema({
   }
 });
 
+// queries
```

Comment:
See the "Query Helpers" section of the mongoose schema docs:
http://mongoosejs.com/docs/guide.html